src/main/java/com/bitwig/extensions/controllers/novation/launchpad_pro/StopClipOverlay.java

```diff
@@ -3,27 +3,25 @@
 import com.bitwig.extension.controller.api.Track;
 import com.bitwig.extension.controller.api.TrackBank;
 
-final class StopClipOverlay extends Overlay
-{
-   StopClipOverlay(final LaunchpadProControllerExtension driver)
-   {
+final class StopClipOverlay extends Overlay {
+   StopClipOverlay(final LaunchpadProControllerExtension driver) {
       super(driver, "stop-clip");
 
       final TrackBank trackBank = driver.mTrackBank;
-      for (int x = 0; x < 8; ++x)
-      {
+      for (int x = 0; x < 8; ++x) {
          final Track track = trackBank.getItemAt(x);
          final Button button = driver.getPadButton(x, 0);
          bindPressed(button, () -> {
             final boolean useAlt = driver.isShiftOn();
-            if (useAlt)
-               track.stopAlt();
-            else
-               track.stop();
+//            if (useAlt)
```

Comment:
Why?

Reply:
That was a mistake. Accidentally checked that one in. It was only meant for me locally.